### DISCUSSION TITLE
Wizard thunder spell doesnt stun when shot

### DISCUSSION THREAD

**Discussion by Leander-0** [unitystation/unitystation#8198] [Ideas]
### Environment
- Game Version: Staging 21110423
- OS Version: Arch Linux

### Detailed description of bug

when the wizard casts thunder spell if deals damage but doesnt stun the player, could be possible that the tesla energy ball has this same issue.

https://user-images.githubusercontent.com/80334192/140855609-696855a7-8a9f-4475-b74c-2e2eb9162106.mp4

### Steps to reproduce

1. spawn spell bug 
2. shoot someone/yourself with thunder
3. damage is done but no stun

**Comment by PerfectTangent** [comment]
I actually don't think it's supposed to. There are two lightning spells in /tg/. One stuns, the other, this version, doesn't. I'm open to hearing arguments that it _should_ stun, though, so I am going to convert this to a discussion. It doesn't seem like a terrible idea to consolidate the two lightning spells into one, though it may require rebalancing how much damage this spell does.

**Reply by AtnerNT** [reply]
I just think its a bit unsatisfying on the players end to shot this huge bolt of lightning and it doesnt even stun them. Maybe keep both spells, but make the stun one have a much smaller VFX so it feels weaker and make it have a faster cool down, whereas the one shown in the video would stun and deal damage but be on a much longer cooldown. Like 'Great lightning bolt' and 'Lesser lightning bolt' spell
